tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_docker.py

```diff
@@ -58,6 +60,8 @@ def __init__(self, *args, **kwargs):
             time.sleep(self.delay)
             self.agent_entity = ALEntityDocker(self.name, is_controller=False)
 
+        self.wired_sniffer = Sniffer(_get_bridge_interface(self.docker_network),
```

Comment:
This is not correct - there should be a single, global, wired sniffer.

But then the problem, of course, is that you don't know the docker_network at the global scope.

So I think the solution is to only define the sniffer in the DUT specification. I'm not sure, however, how to distinguish the top-level device from the others... Maybe with an additional, explicit option in the JSON file? So in the JSON file we'd set `"wired_interface": "eth0"` to enable the sniffer on eth0. And we could have a special case `"wired_interface": "docker"` to use `self.docker_network`.